docs/src/components/RightSidebar/ThemeToggleButton.tsx

```diff
@@ -1,6 +1,7 @@
 import type { FunctionalComponent } from 'preact';
 import { h, Fragment } from 'preact';
 import { useState, useEffect } from 'preact/hooks';
+import './ThemeToggleButton.css';
 
 const themes = ['light', 'dark'];
 
```

Comment:
I'm not sure this is a comment for this PR, but I'd really appreciate it if we had a way to unset your theme preference and just use "prefers-color-scheme" for the theme switcher, since I often go to sites and fiddle with the theme to check out the colors, but then I just want it to follow my system after that!